build/Targets/Settings.props

```diff
@@ -71,6 +71,9 @@
 
     <RoslynNetSdkRootPath>$(MSBuildSDKsPath)\Microsoft.NET.Sdk\Sdk\</RoslynNetSdkRootPath> 
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+
+    <!-- We should be able to remove this extra property once Roslyn projects include the Sdk="Microsoft.NET.Sdk" attribute. -->
```

Comment:
All that said, I also found that the following is a supported syntax for including an SDK:

```<Import SDK="Microsoft.NET.Sdk" Project="Sdk.props" />```

This means that it's somewhat more likely that someone might customize their SDK projects in a similar way to Roslyn.  I've opened an issue for Live Unit Testing (issue 1949, for those with access) to consider whether there's a comprehensive solution for projects that do a "late" import of Microsoft.Common.props.